src/config.js

```diff
@@ -17,49 +17,49 @@ export default {
     '!drop': {
       emoteMultiplier: 1,
       velocities: {
-        min: 3,
-        max: 7,
+        min: Math.floor(Math.random() * 5) + 3,
```

Comment:
The min here is between 3 and 8
The max here is between 5 and 12

So a chance that your max could be lower than your min.